pkgs/applications/networking/syncplay/trusted_certificates.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/syncplay/client.py b/syncplay/client.py
```

Comment:
Is this patch upstreamable?

Reply:
possibly? but if I were upstream, I probably wouldn't accept it; it's only needed because of a patch we have to certifi in nixpkgs (see https://github.com/NixOS/nixpkgs/pull/238091)